backend/django_core/static_src/functions/tailwind.ts

```diff
@@ -0,0 +1,5 @@
+// Code from : https://stackoverflow.com/a/59982143
+import resolveConfig from "tailwindcss/resolveConfig";
+import tailwindConfig from "$tailwind";
+
+export default resolveConfig(tailwindConfig);
```

Comment:
This should be modified, we dont want the entire tailwind payload in JS object

Reply:
This will be fixed on next PR.
I hope so :")